src/System.Console/tests/NegativeTesting.cs

```diff
@@ -8,93 +8,54 @@
 
 public class NegativeTesting
 {
-    [Fact]
-    [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)] // In appcontainer, the stream cannot be opened: there is no Console
-    public static void OpenStandardOutNegativeTests()
-    {
-        NegativeConsoleOutputTests(Console.OpenStandardOutput());
-    }
-
-    [Fact]
-    [SkipOnTargetFramework(~TargetFrameworkMonikers.Uap)] // In appcontainer, the stream cannot be opened: there is no Console
+    [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsWinRT))] // In appcontainer, the stream cannot be opened: there is no Console
```

Comment:
Being in an appcontainer is a runtime condition, not a build configuration condition. I don't think SOTFM really fits here.